### PR TITLE
Add Project Packing Request Review Integration Test

### PR DESCRIPTION
**Pull request by Lee-JaeHyuk**
## Description
Add Project Packing Request Review Integration test

## Type of change
Please insert 'x' one of the type of change.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by astrod — CHANGES_REQUESTED** [review]
```
    @Test
    void testOssNotice() throws Exception {
        mockMvc.perform(post("/project/verification/saveAjax")
                        .content(EntityUtils.toString(new UrlEncodedFormEntity(Arrays.asList(
                                new BasicNameValuePair("prjId", prj.getPrjId()),
                                new BasicNameValuePair("noticeType", "10"),
                                new BasicNameValuePair("appended", ""),
                                new BasicNameValuePair("appendedTEXT", ""),
                                new BasicNameValuePair("networkServerFlag", "N"),
                                new BasicNameValuePair("setPackageJson", ""),
                                new BasicNameValuePair("setPackageFileId", ""),
                                new BasicNameValuePair("setEditCompanyYn", "Y")
                        ))))
                        .param("useCustomNoticeYn", "N")
                        .param("allowDownloadNoticeHTMLYn", "Y")
                        .param("allowDownloadNoticeTextYn", "N")
                        .param("allowDownloadSimpleHTMLYn", "N")
                        .param("allowDownloadSimpleTextYn", "N")
                        .param("allowDownloadSPDXSheetYn", "N")
                        .param("allowDownloadSPDXRdfYn", "N")
                        .param("allowDownloadSPDXTagYn", "N")
                        .header("user-agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.107 Safari/537.36")
                        .contentType(MediaType.APPLICATION_FORM_URLENCODED))
                .andExpect(status().isOk());
```
Fix the test code like it

**Review by astrod — APPROVED** [review]

File: src/test/java/oss/fosslight/controller/PackingRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        pr = (Project) mockMvc.perform(MockMvcRequestBuilders.get("/project/edit/" + prj.getPrjId())
+                .header("user-agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.107 Safari/537.36"))
+                .andExpect(status().isOk())
+                .andReturn().getModelAndView().getModel().get("project");
+        return pr;
+    }
+
+    @Test
+    @DisplayName("Packing request review should be success When parameter is valid")
+    void PackingRequestReviewSuccess() throws Exception{
```
Comment (astrod) [inline src/test/java/oss/fosslight/controller/PackingRequestTest.java:122]: change uppercase to lowercase at the first letter

File: src/test/java/oss/fosslight/controller/PackingRequestTest.java [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                        Assertions.assertThat(editProject(prj.getPrjId()).getVerificationStatus()).isEqualTo("REQ")
+                );
+
+        //then
+
+    }
+
+    @Test
+    @DisplayName("Packing request review should be fail When parameter is invalid")
+    void PackingRequestReviewFail() throws Exception{
```
Comment (astrod) [inline src/test/java/oss/fosslight/controller/PackingRequestTest.java:168]: change uppercase to lowercase at the first letter

Comment (Lee-JaeHyuk) [reply src/test/java/oss/fosslight/controller/PackingRequestTest.java:168]: i change it

**Pull request merged**